web/styles/compose.css

```diff
@@ -426,6 +426,19 @@
                 color: hsl(204deg 49% 29% / 75%);
             }
         }
+
```

Comment:
The style changes in this commit should actually be combined with the previous commit. It's important that if we merged any subset of these commits, the code would be in a fully functional state. So you want the style for the button to be in the same commit as where it needs to be used :)